File: finah-desktop/src/finah_desktop/LeeftijdsCategorie.java
package finah_desktop;

public class LeeftijdsCategorie {

	private int leeftijdsCategorieId;
	private short van;
	private short tot;

	public LeeftijdsCategorie() {

	}

	public LeeftijdsCategorie(short van, short tot) {
		this.van = van;
		this.tot = tot;
	}

	public int getLeeftijdsCategorieId() {
		return leeftijdsCategorieId;
	}

	public void setLeeftijdsCategorieId(int leeftijdsCategorieId) {
		this.leeftijdsCategorieId = leeftijdsCategorieId;
	}

	public short getVan() {
		return van;
	}

	public void setVan(short van) {
		this.van = van;
	}

	public short getTot() {
		return tot;
	}

	public void setTot(short tot) {
		this.tot = tot;
	}
}

File: finah-web/PHP/Models/LeeftijdsCategorie.php
<?php

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 24/03/2015
     * Time: 15:58
     */
    class LeeftijdsCategorie
    {
        public $Id;
        public $Van;
        public $Tot;

        public function __construct($id = null, $Van = null, $Tot = null)
        {
            $this->Id = $id;
            $this->Van = $Van;
            $this->Tot = $Tot;
        }

        public function getId()
        {
            return $this->Id;
        }

        public function setId($Id)
        {
            $this->Id = $Id;
        }

        public function getVan()
        {
            return $this->Van;
        }

        public function setVan($Van)
        {
            $this->Van = $Van;
        }

        public function getTot()
        {
            return $this->Tot;
        }

        public function setTot($Tot)
        {
            $this->Tot = $Tot;
        }
    }